examples/petsc/area.c

```diff
@@ -60,24 +60,24 @@ int main(int argc, char **argv) {
   PetscInt ierr;
   MPI_Comm comm;
   char filename[PETSC_MAX_PATH_LEN],
-       ceedresource[PETSC_MAX_PATH_LEN] = "/cpu/self";
-  PetscInt lsize, gsize, xlsize,
-           qextra  = 1, // default number of extra quadrature points
-           ncompx  = 3, // number of components of 3D physical coordinates
-           ncompu  = 1, // dimension of field to which apply mass operator
-           topodim = 2, // topological dimension of manifold
-           degree  = 3; // default degree for finite element bases
+       ceed_resource[PETSC_MAX_PATH_LEN] = "/cpu/self";
+  PetscInt l_size, g_size, xl_size,
+           q_extra  = 1, // default number of extra quadrature points
+           ncomp_x  = 3, // number of components of 3D physical coordinates
```

Comment:
How about `n_comp_x`?

Reply:
I'd be fine using `num_comp_x` (I think that's how we have it in Rust), but I'd prefer to avoid `n_comp_x`. The later feels harder to read to me.

Reply:
I see your point. Yes, I like `num_comp_x`.

Reply:
Thanks for asking - I went ahead and make some changes in that vein

Reply:
Thank you for taking care of that. I will make the same changes in the fluids example.